docs/generic/helpdesk.md

```diff
@@ -12,9 +12,9 @@ hide:
 
 # Help Desk
 
-The LUMI User Support Team is here to help if you have any questions or problems
-regarding to your use of LUMI. You can find the contact forms to reach us with different issues by clicking the 'Contact us' button:
```

Comment:
Emmanuel wished that it would be clear that by clicking the 'contact us' one would get to the contact forms

Reply:
I will add this extra note about contact form again.